server/src/server.ts

```diff
@@ -2,4 +2,5 @@ import express from 'express';
 import { createServer } from 'http';
 
 export const app = express();
+app.use(express.json());
```

Comment:
make more sense to have this in index.ts around line 18

Reply:
Resolved.